### DISCUSSION TITLE
大佬可以优化一下自定义规则方式吗

### DISCUSSION THREAD

**Discussion by yzxjoe** [gkd-kit/gkd#477] [Q&A] [answered]
**普通用户从零开始学习成本太大了，研究了一晚上，还是不会，要爆炸啦！！T_T
类似李跳跳的编写难度那样就很好
ps:执行选择器可以点走广告的，到修改内存订阅这一步就出错了，太难了...可以帮我看看是哪里出错了吗**

`{
id:'io.dcloud.PandoraEntryActivity',
name:'漫蛙',
groups:[
{key:1,
name:'去广告'
activityIds: ['io.dcloud.PandoraEntryActivity'],
rules: [
        {
          matches: '[text='略过']',
snapshotUrls:'https://i.gkd.li/snapshot/1704977314156',
}
],
},
],
}`

**Comment by lisonge** [comment, marked as answer]
把 略过 两边的单引号改为双引号

你的 id 也是错误的

此外你的 json 也是错误的，`name:'去广告' activityIds` 中间少了逗号

---

> 类似李跳跳的编写难度那样就很好

按照你的这种错误写法，这个规则在上述应用仍然是错误

---

可以在审查工具使用选择器得到查询结果后点击复制图标得到一个简单的规则

**Reply by yzxjoe** [reply]
wc，回复真快！
试了一下还是出错
![Uploading 360截图20240111235950249.jpg…]()

**Comment by yzxjoe** [comment]
大师我悟了！！！你这样说就比想象中简单很多了
